Makefile

```diff
@@ -47,3 +50,25 @@ lint: install-lint  ## Run linter to check formatting and style
 
 format: install-lint  ## Run standard python formatting
 	./code-quality/format ${LINT_PATHS}
+
+
+# Minimal makefile for Sphinx documentation
+
+# The following is auto-generated by sphinx-quickstart: 
+# To test out doc changes locally, run 
+# poetry run make html && open build/html/index.html
+SPHINXOPTS    ?=
+SPHINXBUILD   ?= sphinx-build
+SOURCEDIR     = sphinx_docs
+BUILDDIR      = build
+
+# Put it first so that "make" without argument is like "make help".
+help:
+	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+
+.PHONY: help Makefile
+
+# Catch-all target: route all unknown targets to Sphinx using the new
```

Comment:
Does this mean if you type `make foobartypo` for the SDK it launches sphinx for something?  I'm not sure why we'd want that - seems like it would be more confusing than helpful.

Reply:
This was autogenerated, but I also think it's more confusing than helpful, so removing it!